src/offline/cable_checks.F90

```diff
@@ -50,141 +52,561 @@ MODULE cable_checks_module
+
+     CALL  check_range("canst1", veg%canst1, ranges%canst1, ktau, met)
+     CALL  check_range("dleaf", veg%dleaf, ranges%dleaf, ktau, met)
+     CALL  check_range("ejmax", veg%ejmax, ranges%ejmax, ktau, met)
+     CALL  check_range("meth", veg%meth, ranges%meth, ktau, met)
+     CALL  check_range("frac4", veg%frac4, ranges%frac4, ktau, met)
+     CALL  check_range("hc", veg%hc, ranges%hc, ktau, met)
+     ! CALL  check_range("vlai", veg%vlai, ranges%vlai, ktau, met)
+     CALL  check_range("xalbnir", veg%xalbnir, ranges%xalbnir, ktau, met)
+     CALL  check_range("rp20", veg%rp20, ranges%rp20, ktau, met)
+     CALL  check_range("rpcoef", veg%rpcoef, ranges%rpcoef, ktau, met)
+     CALL  check_range("rs20", veg%rs20, ranges%rs20, ktau, met)
+     CALL  check_range("shelrb", veg%shelrb, ranges%shelrb, ktau, met)
+     CALL  check_range("vegcf", veg%vegcf, ranges%vegcf, ktau, met)
+     CALL  check_range("tminvj", veg%tminvj, ranges%tminvj, ktau, met)
+     ! CALL  check_range("toptvj", veg%toptvj, ranges%toptvj, ktau, met)
+     CALL  check_range("tmaxvj", veg%tmaxvj, ranges%tmaxvj, ktau, met)
+     CALL  check_range("vbeta", veg%vbeta, ranges%vbeta, ktau, met)
+     CALL  check_range("vcmax", veg%vcmax, ranges%vcmax, ktau, met)
+     CALL  check_range("xfang", veg%xfang, ranges%xfang, ktau, met)
+     CALL  check_range("extkn", veg%extkn, ranges%extkn, ktau, met)
+     ! CALL  check_range("vlaimax", veg%vlaimax, ranges%vlaimax, ktau, met)
+     CALL  check_range("wai", veg%wai, ranges%wai, ktau, met)
+     ! CALL  check_range("a1gs", veg%a1gs, ranges%a1gs, ktau, met)
+     ! CALL  check_range("d0gs", veg%d0gs, ranges%d0gs, ktau, met)
+     ! CALL  check_range("alpha", veg%alpha, ranges%alpha, ktau, met)
+     ! CALL  check_range("convex", veg%convex, ranges%convex, ktau, met)
+     ! CALL  check_range("cfrd", veg%cfrd, ranges%cfrd, ktau, met)
+     ! CALL  check_range("gswmin", veg%gswmin, ranges%gswmin, ktau, met)
+     ! CALL  check_range("conkc0", veg%conkc0, ranges%conkc0, ktau, met)
+     ! CALL  check_range("conko0", veg%conko0, ranges%conko0, ktau, met)
+     ! CALL  check_range("ekc", veg%ekc, ranges%ekc, ktau, met)
+     ! CALL  check_range("eko", veg%eko, ranges%eko, ktau, met)
+     CALL  check_range("g0", veg%g0, ranges%g0, ktau, met)
+     CALL  check_range("g1", veg%g1, ranges%g1, ktau, met)
+
+     !!! Real r_2
+
+     CALL  check_range("rootbeta", veg%rootbeta, ranges%rootbeta, ktau, met)
+     ! CALL  check_range("gamma", veg%gamma, ranges%gamma, ktau, met)
+     ! CALL  check_range("ZR", veg%ZR, ranges%ZR, ktau, met)
+     ! CALL  check_range("F10", veg%F10, ranges%F10, ktau, met)
+     ! CALL  check_range("clit", veg%clit, ranges%clit, ktau, met)
+
+     !! 2D
+
+     !!! logical,	public,	DIMENSION(:, ktau, met), POINTER	::	deciduous		! Review: No need of this since T/F	
+
+     !!! Integer
+     ! CALL  check_range("disturbance_interval", veg%disturbance_interval, ranges%disturbance_interval, ktau, met)
+
+     !!! Real
```

Comment:
```suggestion
     !~ 2D

     !~~ logical,	public,	DIMENSION(:, ktau, met), POINTER	::	deciduous		! Review: No need of this since T/F	

     !~~ Integer
     ! CALL  check_range("disturbance_interval", veg%disturbance_interval, ranges%disturbance_interval, ktau, met)

     !~~ Real
```